src/xpHandling.js

```diff
@@ -37,6 +37,14 @@ async function addXp(member, xp, silent = false) {
                 .catch((e) => console.error('Error sending level down dm!', e));
         }
     }
+
+    // Ensure recent xp.
+    if (userDto.recentReset < Date.now()) {
+        userDto.activityRecent = 0;
+        userDto.recentReset = getTomorrow();
```

Comment:
Hmh, why would this be per user and not global?
Also kinda weird naming for recentReset to be in the future.

Reply:
So I don't have to update every users in the database every 24 hours. RecentRecwt should probably be better named. It is when I reset the user's recent value. Basically of past this value it's activity recent is Invalid.

Reply:
activityValidUntil might be a better name